app/src/main/java/com/keylesspalace/tusky/usecase/TimelineCases.kt

```diff
@@ -95,30 +94,22 @@ class TimelineCases @Inject constructor(
         }
     }
 
-    fun mute(statusId: String, notifications: Boolean, duration: Int?) {
-        mastodonApi.muteAccount(statusId, notifications, duration)
-            .subscribe(
-                {
-                    eventHub.dispatch(MuteEvent(statusId))
-                },
-                { t ->
-                    Log.w("Failed to mute account", t)
-                }
-            )
-            .addTo(cancelDisposable)
+    suspend fun mute(statusId: String, notifications: Boolean, duration: Int?) {
+        try {
+            mastodonApi.muteAccount(statusId, notifications, duration)
+            eventHub.dispatch(MuteEvent(statusId))
+        } catch (t: Throwable) {
+            Log.w("Failed to mute account", t)
```

Comment:
Ha, no. This retains a bug in the original code, the first parameter should be the log tag.